README.md

```diff
@@ -1,67 +1,51 @@
-| Python + FlatBuffers  | `foxglove-schemas-flatbuffer` | [![pypi version](https://shields.io/pypi/v/foxglove-schemas-flatbuffer)](https://pypi.org/project/foxglove-schemas-flatbuffer/)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
-| ROS                   | `foxglove_msgs`               | [![ROS Melodic version](https://img.shields.io/ros/v/melodic/foxglove_msgs)](https://index.ros.org/p/foxglove_msgs/github-foxglove-schemas/#melodic) [![ROS Noetic version](https://img.shields.io/ros/v/noetic/foxglove_msgs)](https://index.ros.org/p/foxglove_msgs/github-foxglove-schemas/#noetic) [![ROS Foxy version](https://img.shields.io/ros/v/foxy/foxglove_msgs)](https://index.ros.org/p/foxglove_msgs/github-foxglove-schemas/#foxy) [![ROS Galactic version](https://img.shields.io/ros/v/galactic/foxglove_msgs)](https://index.ros.org/p/foxglove_msgs/github-foxglove-schemas/#galactic) [![ROS Humble version](https://img.shields.io/ros/v/humble/foxglove_msgs)](https://index.ros.org/p/foxglove_msgs/github-foxglove-schemas/#humble) [![ROS Iron version](https://img.shields.io/ros/v/iron/foxglove_msgs)](https://index.ros.org/p/foxglove_msgs/github-foxglove-schemas/#iron) [![ROS Rolling version](https://img.shields.io/ros/v/rolling/foxglove_msgs)](https://index.ros.org/p/foxglove_msgs/github-foxglove-schemas/#rolling) |
+Visit [Foxglove Docs](https://docs.foxglove.dev/) to get started.
 
-## Introduction
+## Getting started
 
-See [Foxglove Schemas documentation](https://docs.foxglove.dev/docs/visualization/message-schemas/introduction).
+### Python
```

Comment:
Why does python not get an "alpha" quality label? Really the whole SDK should have an "alpha" label call out

Reply:
I have not listed the python sdk, since it is not yet ready for testing. Only the existing ws-protocol library is listed here.

Reply:
removed rust sdk for now as well, for consistency. we can list it when stable.